benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -513,10 +498,9 @@ msgid "core.pages.agency_index.title"
 msgstr "Introduction"
 
 #: benefits/core/views.py:63
-msgid "core.pages.agency_index.content_title"
+msgid "core.pages.agency_index.mst_cc.content_title"
```

Comment:
I think this rename needs to be done in the Spanish .po file too.

Btw, you can run `./bin/makemessages.sh` to regenerate the .po files. It'll update line number references, add in missing entries, and remove any unused entries! 

... typing that out just made me realize that we don't have documentation on this! I created #954 to fix that.